README.md

```diff
@@ -38,7 +38,7 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+``` Scope is what you are working with lik the objects, functions, variables.
```

Comment:
Please review.  I recommend you review the answer in the solution branch as well as review this http://stackoverflow.com/questions/500431/what-is-the-scope-of-variables-in-javascript.